src/qml/SettingsPage.qml

```diff
@@ -0,0 +1,45 @@
+/*
+ *   Copyright 2019 Ildar Gilmanov <gil.ildar@gmail.com>
+ *   This file based on sample code from Kirigami
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU Library General Public License as
+ *   published by the Free Software Foundation; either version 3, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU Library General Public License for more details
+ *
+ *   You should have received a copy of the GNU Library General Public License
+ *   along with this program; if not, see <https://www.gnu.org/licenses/>
+ */
+
+import QtQuick 2.7
+import QtQuick.Controls 2.4 as QQC2
+import org.kde.kirigami 2.6 as Kirigami
+import QtQuick.Layouts 1.11
+
+Kirigami.ScrollablePage {
+    objectName: "settingsPage";
+    title: qsTr("Settings");
+
+    Kirigami.AbstractCard {
+        width: parent.width;
+        header: Kirigami.Heading {
+            text: qsTr("Use Auto-reconnect if your tail drops the connection?");
```

Comment:
This will want to be changed at some point (using Cards for this seems very strange), but... well, it works.

Reply:
Just only one example of the check box view style at the application that I found at the `WelcomeView`, and that uses `AbstactCard`. I think it is not good to use it for checkboxes too, but...

Any way when we will be adding new fields at the `SettingsPage` we can use another approach

Reply:
Yeah, it's fine for now - it's larger than using the listview style used on the Casual Mode settings page, but given the small amount of content currently on the main settings page, using a Card isn't that bad :)